src/js/view/preview.js

```diff
@@ -50,6 +50,20 @@ define([
             var backgroundImage = '';
             if (_.isString(img)) {
                 backgroundImage = 'url("' + img + '")';
+
+                // Save the background image's width and height for resize check in view.js
+                if (this.image) {
+                    this.image.onload = null;
+                }
```

Comment:
This should be moved to top of setImage. We still need to do it if 'img' is null or not a string.